x/hard/client/cli/query.go

```diff
@@ -272,10 +309,42 @@ func queryBorrowsCmd(queryRoute string, cdc *codec.Codec) *cobra.Command {
 				owner = borrowOwner
 			}
 
+			// Execute 'borrow' query: queries a user's borrow balance, including their outstanding debt
+			if !owner.Empty() {
+				params := types.NewQueryBorrowParams(owner)
+				bz, err := cdc.MarshalJSON(params)
+				if err != nil {
+					return err
+				}
+
+				route := fmt.Sprintf("custom/%s/%s", queryRoute, types.QueryGetBorrow)
+				res, height, err := cliCtx.QueryWithData(route, bz)
+				if err != nil {
+					return err
+				}
+				cliCtx = cliCtx.WithHeight(height)
+
+				var balance sdk.Coins
+				if err := cdc.UnmarshalJSON(res, &balance); err != nil {
+					return fmt.Errorf("failed to unmarshal borrow balance: %w", err)
+				}
+
+				if len(denom) > 0 {
+					if balance.AmountOf(denom).Equal(sdk.ZeroInt()) {
+						return fmt.Errorf("user %s has no borrow balance for denom %s", owner, denom)
+					}
+				}
+
+				return cliCtx.PrintOutput(balance)
```

Comment:
Same as deposits, return the object not the balance

Reply:
Updated in https://github.com/Kava-Labs/kava/pull/761/commits/e4bd838873c7cb543d33bd8cfa4dfb7f77e8e751